vendor/assets/javascripts/jquery.textile.js

```diff
@@ -192,8 +192,10 @@
       $.post(this.$element.data('preview-url'),
         data,
         function(result) {
-          this.options.$preview.removeClass('loading-indicator')
-            .html(result);
+          this.options.$preview.removeClass('loading-indicator').html(result);
+          if (result == '\n') {
+            this.options.$preview.append('<div class="preview-placeholder"><h5>Add some fields to see a live preview here</h5></div>')
```

Comment:
Does the default state on line 32 not need this change as well?
Actually the entire loading-indicator stuff- does it do anything? Can we pull that out in favour of the blank slate work being applied.

If we need the blank slate in two places we'll probably have to pull that content out into a function or variable to utilize in both places.

Reply:
@brianp 
> Does the default state on line 32 not need this change as well?
Nope. Loading happens first, then the preview is rendered (blank or otherwise).

> Actually the entire loading-indicator stuff- does it do anything?
Yes, it shows a loading indicator while loading is happening, once the loading done the indicator is hidden and the content is displayed. If there is no content the placeholder is rendered.

> Can we pull that out in favour of the blank slate work being applied.
No, because the loading needs to happen first. If we take that out then when slow loading happens the user will be told to add fields when really fields do exist but just haven't been rendered yet. It would cause a bad/confusing user experience.

> If we need the blank slate in two places
It is not. We only need it here in `_loadPreview` because that is executed on load and on change so it executes in both situations where it is needed. 

The flow is: 
1. Attempt to load the pane and show the loading indicator while it is loading.
2. Once it is loaded show the content and remove the loading indicator.
3. If there is no content, show the placeholder.